### ISSUE TITLE
ConnectionPoolMetricsTests occasionally fail with NaN.

### ISSUE THREAD

**Issue by mp911de** [spring-projects-experimental/spring-boot-r2dbc#31]
```
org.junit.ComparisonFailure: 
Expected :7.0
Actual   :NaN
<Click to see difference>


	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at org.springframework.boot.actuate.r2dbc.metrics.ConnectionPoolMetricsTests.assertGauge(ConnectionPoolMetricsTests.java:91)
	at org.springframework.boot.actuate.r2dbc.metrics.ConnectionPoolMetricsTests.doTest(ConnectionPoolMetricsTests.java:85)
	at org.springframework.boot.actuate.r2dbc.metrics.ConnectionPoolMetricsTests.metrics(ConnectionPoolMetricsTests.java:61)
```

or

```
org.junit.ComparisonFailure: expected:<[3.0]> but was:<[NaN]>
	at org.springframework.boot.actuate.r2dbc.metrics.ConnectionPoolMetricsTests.assertGauge(ConnectionPoolMetricsTests.java:83)
	at org.springframework.boot.actuate.r2dbc.metrics.ConnectionPoolMetricsTests.metrics(ConnectionPoolMetricsTests.java:74)
```

**Comment by mp911de** [comment]
See also #17.

**Comment by mp911de** [comment]
The fix is to upgrade to the latest build of 0.8.0 snapshots.

**Comment by mp911de** [comment]
Fixed via #34.
